jobs/updatenetwork.rb

```
@@ -1,134 +1,10 @@
-       #network_debug.puts "counter = #{counter}"
-       #network_debug.puts "line[0] = #{line[0]}"
-       #while points[counter][:x] != line[0].to_f do
-       #  network_debug.puts "Incrementing counter #{points[counter][:x]} != #{line[0].to_f}"
-       #  if 
-       #  counter += 1
-       #end
-       #if points[counter][:x] == line[0].to_f 
-       #  network_debug.puts "Found an equal: #{points[counter][:x]} = #{line[0].to_f}"
-       #else
-       #  next
-       #end
-       #network_debug.puts "Before add: #{points[counter][:y]}"
-       points[counter][:y] += ( line[1].to_f + line[2].to_f)*8
-       #network_debug.puts "After add: #{points[counter][:y]}"
-       last_point = points[counter][:y]
-       counter += 1
-     end
-  end
-
-  counter = 0
-  rrd3.fetch(:average).each do |line|
-     if line[1].class != String && line[1].nan? != true
-       if min_point == 0
-         min_point = line[0].to_f
-       end
-       #while points[counter][:x] != line[0].to_f do
-       #  #network_debug.puts "Incrementing counter #{points[counter][:x]} != #{line[0].to_f}" 
-       #  counter += 1
-       #end
-       #if points[counter][:x] == line[0].to_f 
-         #network_debug.puts "Found an equal: #{points[counter][:x]} = #{line[0].to_f}"
-       #else
-       #  next
-       #end
-       #network_debug.puts "Before add: #{points[counter][:y]}"
-       points[counter][:y] += ( line[1].to_f + line[2].to_f)*8
-       #network_debug.puts "After add: #{points[counter][:y]}"
-       last_point = points[counter][:y]
-       counter += 1
-     end
-  end
-  
-  #network_debug.close
-
-  send_event('NetworkGraph', { points: points })
-  send_event('HCCAmazonPrice', { network_bandwidth: last_point })
+(1..30).each do |i|
+    send_event('ShorNetworkGraph', { points: points}  )
+    send_event('PkiNetworkGraph', { points: points}  )
+    send_event('HCCAmazonPrice', { network_bandwidth: 1000000000000000000000000000000 })
+    sleep(0.2)
```

Wait, so there is no actual network data?  It's all faked?

---

This is initial dummy data that was put in to try to fix the graph generating error. This is no longer an issue with the rework.